awebox/mdl/aero/indicators.py

```diff
@@ -80,138 +87,149 @@ def get_performance_outputs(options, atmos, wind, variables, outputs, parameters
 
-    outputs['aerodynamics']['f_aero' + str(n)] = f_aero
-    outputs['aerodynamics']['f_lift' + str(n)] = f_lift
-    outputs['aerodynamics']['f_drag' + str(n)] = f_drag
-    outputs['aerodynamics']['f_side' + str(n)] = f_side
+    outputs['aerodynamics']['air_velocity' + str(kite)] = air_velocity
+    airspeed = vect_op.norm(air_velocity)
+    outputs['aerodynamics']['airspeed' + str(kite)] = airspeed
+    outputs['aerodynamics']['u_infty' + str(kite)] = wind.get_velocity(q[2])
 
-    outputs['aerodynamics']['ehat_chord' + str(n)] = ehat_chord
-    outputs['aerodynamics']['ehat_span' + str(n)] = ehat_span
-    outputs['aerodynamics']['ehat_up' + str(n)] = vect_op.normed_cross(ehat_chord, ehat_span)
+    rho = atmos.get_density(q[2])
+    outputs['aerodynamics']['air_density' + str(kite)] = rho
+    outputs['aerodynamics']['dyn_pressure' + str(kite)] = 0.5 * rho * cas.mtimes(air_velocity.T, air_velocity)
 
-    b_ref = parameters['theta0', 'geometry', 'b_ref']
-    c_ref = parameters['theta0', 'geometry', 'c_ref']
-    gamma_cross = vect_op.norm(f_lift) / b_ref / rho / vect_op.norm(vect_op.cross(ua, ehat_span))
-    gamma_cl = 0.5 * ua_norm**2. * aero_coefficients['CL'] * c_ref / vect_op.norm(vect_op.cross(ua, ehat_span))
-    outputs['aerodynamics']['gamma_cross' + str(n)] = gamma_cross
-    outputs['aerodynamics']['gamma_cl' + str(n)] = gamma_cl
-    outputs['aerodynamics']['gamma' + str(n)] = gamma_cl
+    ehat_chord = kite_dcm[:, 0]
+    ehat_span = kite_dcm[:, 1]
+    ehat_up = kite_dcm[:, 2]
 
-    outputs['aerodynamics']['wingtip_ext' + str(n)] = q + ehat_span * b_ref / 2.
-    outputs['aerodynamics']['wingtip_int' + str(n)] = q - ehat_span * b_ref / 2.
+    outputs['aerodynamics']['ehat_chord' + str(kite)] = ehat_chord
+    outputs['aerodynamics']['ehat_span' + str(kite)] = ehat_span
+    outputs['aerodynamics']['ehat_up' + str(kite)] = ehat_up
 
-    outputs['aerodynamics']['fstar_aero' + str(n)] = cas.mtimes(ua.T, ehat_chord) / c_ref
+    outputs['aerodynamics']['f_aero_body' + str(kite)] = f_aero_body
+    outputs['aerodynamics']['f_aero_control' + str(kite)] = f_aero_control
+    outputs['aerodynamics']['f_aero_earth' + str(kite)] = f_aero_earth
+    outputs['aerodynamics']['f_aero_wind' + str(kite)] = f_aero_wind
 
-    outputs['aerodynamics']['r'+str(n)] = r.reshape((9,1))
+    ortho = cas.reshape(cas.mtimes(kite_dcm.T, kite_dcm) - np.eye(3), (9, 1))
+    ortho_resi = cas.mtimes(ortho.T, ortho)
+    outputs['aerodynamics']['ortho_resi' + str(kite)] = ortho_resi
 
-    if int(options['kite_dof']) == 6:
-        outputs['aerodynamics']['m_aero' + str(n)] = m_aero
 
-    outputs['aerodynamics']['mach' + str(n)] = get_mach(options, atmos, ua, q)
-    outputs['aerodynamics']['reynolds' + str(n)] = get_reynolds(options, atmos, ua, q, parameters)
 
-    return outputs
+    conversion_resis = []
+    conversion_targets = {'control': f_aero_control, 'earth': f_aero_earth, 'wind': f_aero_wind}
```

Comment:
checks to make sure that the conversion between the reference frames (control, earth, wind, and body) are actually working correctly, since we may not be sure in which frame that the stability derivatives are given.. 

these outputs are ultimately used in the quality checks of the solution.

Reply:
ok, thanks!